forge-gui/res/cardsfolder/upcoming/coronation_of_chaos.txt

```diff
@@ -0,0 +1,6 @@
+Name:Coronation of Chaos
+ManaCost:2 R
+Types:Sorcery
+A:SP$ Pump | Cost$ 2 R | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 3 | RememberTargets$ True | KW$ HIDDEN CARDNAME can't block. | IsCurse$ True | TgtPrompt$ Select target creature | SubAbility$ DBGoad | SpellDescription$ Up to three target creatures can't block this turn. Goad them. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
```

Comment:
Remove `Cost$` and `RememberTargets$`

I'd make `TgtPrompt$ Select up to three target creatures`